core/client/fs/src/main/java/alluxio/client/file/FileOutStream.java

```diff
@@ -297,7 +298,8 @@ private void handleCacheWriteException(Exception e) throws IOException {
   protected void scheduleAsyncPersist() throws IOException {
     try (CloseableResource<FileSystemMasterClient> masterClient = mContext
         .acquireMasterClientResource()) {
-      masterClient.get().scheduleAsyncPersist(mUri);
+      masterClient.get().scheduleAsyncPersist(mUri,
+          ScheduleAsyncPersistencePOptions.getDefaultInstance());
```

Comment:
Should these inherit common options?

Reply:
What do you mean?

Reply:
It actually seems a bit weird now that I dig deeper.
I was thinking `FileOutStream` has an `OutStreamOptions` object which has some fields like `TtlAction` that are included in the `CommonOptions` of `ScheduleAsyncPersistencePOptions` so we may want to carry those over.

Reply:
Well for the options we'd need to consolidate some of the fields inside `OutStreamOptions` into `CommonPOptions` but that seems outside of the scope of this PR.

Reply:
Yeah I agree, could you leave a todo/ticket for making sure we propagate options?

Reply:
New issue created